src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -23,6 +23,7 @@ public abstract class SslStreamStreamToStreamTest
         protected abstract bool DoHandshake(SslStream clientSslStream, SslStream serverSslStream);
 
         [Fact]
+        [ActiveIssue(16516, TestPlatforms.Windows)]
```

Comment:
FWIW, this won't actually address https://github.com/dotnet/corefx/issues/17103, which is calling this test explicitly after having enabled the event source, so using ActiveIssue won't suppress it:
https://github.com/dotnet/corefx/blob/02b0fdb699dfc03fd995d278d90f396892095590/src/System.Net.Security/tests/FunctionalTests/LoggingTest.cs#L38